### ISSUE TITLE
Kepler's Laws initial state

### ISSUE THREAD

**Issue by DianaTavares** [phetsims/my-solar-system#79]
In the design meeting (January 4, 2023) we agree the default orbit as circular (similar to the earth-sun behavior), and we are going to include this information in the Teacher's tip document (to let teacher know that if they want to analyze Kepler laws for the earth-sun system, they just need to reset the screen). The sim currently has not zoomed levels, and this present looks small:
![image](https://user-images.githubusercontent.com/26031501/211371094-54b4fdee-4b00-4958-a157-e8027950f20e.png)

The suggestion is included 4 zoom levels. The initial zoom, where the pre-set looks twice bigger that the way it looks now, and add one zoom level to zoom-in, and 2 to zoom-out.

@AgustinVallejo, feel free to play with the zooms levels to see if there is a better fit for this screen. 

Also, this screen needs to change it initial start to open first the first Kepler's Law scene.

**Comment by DianaTavares** [comment]
@AgustinVallejo and I discussed by slack that this pre-set will bring difficulties because the grid is 1AU, then with more zooming, the grid will lose it usability. 
![image](https://user-images.githubusercontent.com/26031501/211377464-386c47da-4f69-439b-a26b-7435f10858e1.png)

We discussed the idea to change the girds to 0.5 AU, but that makes very crowed come zooms options. Also was discussed to have a different grid just for this screen, but that will bring students to not notice that the scaling is different from make mistakes. Probably change the grid in the Kepler's Law simulation, but that is a future discussion. 

We agree that this is the initial state of the Kepler's Law scene: circular orbit at a 2AU distance from the sun for all screens:
 
![image](https://user-images.githubusercontent.com/26031501/211378465-29c03c36-0fa9-4146-96e7-6432d4d9f91a.png)
